<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FriendListItem</class>
 <widget class="QWidget" name="FriendListItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>60</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>250</width>
    <height>60</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>250</width>
    <height>60</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>250</width>
    <height>60</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="picLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="nicknameLabel">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>10</y>
     <width>160</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>160</width>
     <height>20</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>0</width>
     <height>20</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 13px &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>测试0</string>
   </property>
  </widget>
  <widget class="QLabel" name="signLabel">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>30</y>
     <width>160</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>160</width>
     <height>20</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>0</width>
     <height>20</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(125, 125, 125);
font: 12px &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>测试0的签名</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
